src/components/svg/keys/svg-keyboard-key/svg-keyboard-key.component.ts

```diff
@@ -70,6 +70,7 @@ export class SvgKeyboardKeyComponent implements OnInit, OnChanges, OnDestroy {
     @Input() width: number;
     @Input() keyAction: KeyAction;
     @Input() keybindAnimationEnabled: boolean;
+    @Input() popoverEnabled: boolean;
```

Comment:
I am aware that some of the components aren't well designed. Of course those are design mistakes, too. Best would be to create a list and refactor/rename those behaviours/property names. But now all we can do is not introduce more.